### ISSUE TITLE
Negative margin on featured-orgs class and static image

### ISSUE THREAD

**Issue by sonalkr132** [github/government.github.com#403]
Why is .featured-orgs has [negative margin of 55px](https://github.com/github/government.github.com/blob/a668b6cf7b8882406fb250ce91bc7f7dc025d8d0/assets/css/style.scss#L292-#l294)? Images are overflowing the screen because of it:
![screenshot from 2015-09-23 09 54 17](https://cloud.githubusercontent.com/assets/7680662/10037894/48defbf0-61da-11e5-83a1-861b6d65b844.png)

<hr />

Also why are these featured org avatars are stored in repo? Why isn't it using something like (check **src** property):

```
<img class="animate-in" src="https://github.com/{{ org }}.png" width="80" alt="View {{ org }}'s profile on GitHub">
```

Can I make above two changes?

**Comment by benbalter** [comment]
> Why is .featured-orgs has negative margin of 55px?

The intention was to show that there were lots and lots of government organizations on GitHub, and not just those featured. "So many they couldn't fit on the page"

> Also why are these featured org avatars are stored in repo?

IIRC, it was a bandwidth consideration. There is no way to request a particular sized image using the `[login].png` route.

**Comment by sonalkr132** [comment]
> There is no way to request a particular sized image using the [login].png route.

@benbalter  check out #404

~~A follow up question:
Can I please delete images from git history as well?
`.git` dir for this repo is just huge (21MB). There are many unused images in git history and `assets/img/featured-orgs/*` is only going to add up to that. I was having a slow internet day when I first cloned it and it took forever to complete.~~

**EDIT:**
Not a great idea :stuck_out_tongue_closed_eyes:

**Comment by sonalkr132** [comment]
closed with #404
